components/playlist/browser/resources/utils/l10n.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import { getLocale } from '../../../../common/locale'
+
+export type Message =
```

Comment:
It would be nice if we could do this automatically - it's super painful having to type the strings so many times :laughing: 
1. In the grd, the `IDS_UNDERSCORE_NAME` and the English text
2. In the web_ui file `IDS_UNDERSCORE_NAME` and the friendly `braveUnderscoreName`, for use in JS
3. Here, to ensure you use the right message names
4. In the file, where we actually use it :laughing: 

In the interest of typing this out fewer times, I think it's probably fine if you don't provide a type for these imports and just use `getLocale` directly - the WebUI will crash if one of the strings doesn't exist. Up to you though, it's good as is too.

Reply:
Yeah, I feel the same. Let me file an issue for it. https://github.com/brave/brave-browser/issues/32188

> I think it's probably fine if you don't provide a type for these imports and just use getLocale directly - the WebUI will crash if one of the strings doesn't exist

Yeah I agree on that. I think the current implementation has benefit over it bc we can know when we make mistake during the compilation. But these should be a part of the automation, definately.